src/script.sh

```
@@ -224,13 +224,13 @@ main() {
     ### because the next step makes all of the DNANexus
     ### variables I'm using here obsolete
     if [[ "$run_CollectVariantCallingMetrics" == "true" ]]; then
-        if [[ $vcf == *.vcf ]]; then
-            bgzip $vcf
-            tabix -p vcf "${vcf}.gz"
+        if [[ $vcf_name == *.vcf ]]; then
+            bgzip $vcf_path
```

Shellcheck (0.11.0)

[warning] vcf_path is referenced but not assigned.

(SC2154)